membership_file/admin.py

```diff
@@ -87,23 +101,24 @@ def get_export_filename(self, request, queryset, file_format):
 
     fieldsets = [
         (None, {'fields':
-            ['user', ('first_name', 'tussenvoegsel', 'last_name'),
-            'marked_for_deletion',
-            ('last_updated_date', 'last_updated_by'),]}),
+                ['user', ('first_name', 'tussenvoegsel', 'last_name'),
+                 'marked_for_deletion',
+                 ('last_updated_date', 'last_updated_by'), ]}),
         ('Membership Status', {'fields':
-            ['is_deregistered', 'is_honorary_member', 'member_since']}),
+                               ['is_deregistered', 'is_honorary_member', 'member_since']}),
         ('Contact Details', {'fields':
-            ['email', 'phone_number',
-            ('street', 'house_number', 'house_number_addition'), ('postal_code', 'city'), 'country']}),
+                             ['email', 'phone_number',
+                              ('street', 'house_number', 'house_number_addition'), ('postal_code', 'city'), 'country']}),
         ('Room Access', {'fields':
-            ['key_id', 'tue_card_number',
-            ('external_card_number', 'external_card_digits', 'external_card_cluster'),
-            'external_card_deposit', 'accessible_rooms']}),
+                         ['key_id', 'tue_card_number',
+                          ('external_card_number', 'external_card_digits',
+                           'external_card_cluster'),
+                             'external_card_deposit', 'accessible_rooms']}),
         ('Legal Information', {'fields':
-            ['educational_institution', 'student_number',
-            'date_of_birth', 'legal_name']}),
+                               ['educational_institution', 'student_number',
+                                'date_of_birth', 'legal_name']}),
         ('Notes', {'fields':
-            ['notes']}),
+                   ['notes']}),
     ]
```

Comment:
I get where you are going from with the way you place these indices. But personally I think this worsens readability due to its very busy nature. All items that you indented basically have the same function, 'defining availlable fields in a given section' (notice how each line 103, 107, 112, 117, and 120 all end with `'fields':` ). However, due to its variation in indent spaces this is no longer instantly noticeable. Arguably you could move the list brackets outside the selection, but other than that I'd rather see it stay the same.
```
('Legal Information', {'fields':[
   'educational_institution', 'student_number',
   'date_of_birth', 'legal_name'
]}),
```

I do really appreciate you wanting to add code linting and standards into the code. I just believe that this specific implementation here has a counter productive effect

Reply:
If I am honest, I did not think that deeply about it, just a habit of using the formatter at every commit and I think it might have been an accident. I am working on it in a different branch.

I am currently looking at what each formatter makes from it.

`black` thinks this is the best solution: 
<details><summary>Verbose python code</summary>

```python

    fieldsets = [
        (
            None,
            {
                "fields": [
                    "user",
                    ("first_name", "tussenvoegsel", "last_name"),
                    "marked_for_deletion",
                    ("last_updated_date", "last_updated_by"),
                ]
            },
        ),
        (
            "Membership Status",
            {"fields": ["is_deregistered", "is_honorary_member", "member_since"]},
        ),
        (
            "Contact Details",
            {
                "fields": [
                    "email",
                    "phone_number",
                    ("street", "house_number", "house_number_addition"),
                    ("postal_code", "city"),
                    "country",
                ]
            },
        ),
        (
            "Room Access",
            {
                "fields": [
                    "key_id",
                    "tue_card_number",
                    (
                        "external_card_number",
                        "external_card_digits",
                        "external_card_cluster",
                    ),
                    "external_card_deposit",
                    "accessible_rooms",
                ]
            },
        ),
        (
            "Legal Information",
            {
                "fields": [
                    "educational_institution",
                    "student_number",
                    "date_of_birth",
                    "legal_name",
                ]
            },
        ),
        ("Notes", {"fields": ["notes"]}),
    ]
```

</details>

It’s certainly verbose, but it does work… What’s your opinion on it? 

A little of topic and prolly fits better in #13, but I kinda want to go with `black`, ’cause it’s easy to plugin and doesn’t leave much room for error, but it’s very opinionated, and will prolly change a lot of our formatting, at least a lot more than autopep, which I used now, did. Are you fine with that?

Reply:
Otherwise I can restore it to it original state, and I have to look further on formatters.

Reply:
Other problem with black is that it’s not really consistent with fields being one or multiple lines. (For this specific piece of code)

Reply:
Huh, I thought I changed this back.

Reply:
It seems I did?
But then why is it still listed as a diff here?

@DutcherNL Can you check if it's correct?